tests/bluetooth/bsim_bt/bsim_test_mesh/prj.conf

```diff
@@ -4,14 +4,18 @@ CONFIG_SYS_CLOCK_TICKS_PER_SEC=32768
 
 # Bluetooth configuration
 CONFIG_BT=y
-CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_BROADCASTER=y
 CONFIG_BT_PRIVACY=n
+CONFIG_BT_TINYCRYPT_ECC=y
 CONFIG_BT_COMPANY_ID=0x0059
 CONFIG_BT_DEVICE_NAME="Mesh test"
+CONFIG_BT_DEBUG_LOG=y
+
+# Bluetooth Controller
 CONFIG_BT_LL_SW_SPLIT=y
-CONFIG_BT_OBSERVER=y
-CONFIG_BT_BROADCASTER=y
-CONFIG_BT_TINYCRYPT_ECC=y
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_SCAN_UNRESERVED=y
```

Comment:
What is the purpose of explicitly enabling `CONFIG_BT_CTLR_SCAN_UNRESERVED` here? It is already enabled by default for mesh: https://github.com/zephyrproject-rtos/zephyr/blob/cccfc8fc8f627e127cf81b4960c3d2419a477e19/subsys/bluetooth/controller/Kconfig.ll_sw_split#L793-L796.